.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd (somewhat strongly) vote against committing any `.idea` files. I'd be ok adding `/.idea` to the top-level `.gitignore` file.